### ISSUE TITLE
onnxruntime multiclass instructions don't working

### ISSUE THREAD

**Issue by ei-grad** [catboost/catboost#1189]
Problem: example code given in the docs to apply catboost multiclass model via onnxruntime doesn't work
catboost version: 0.21
Operating System: GNU/Linux

onnxruntime==1.1.2

https://catboost.ai/docs/concepts/apply-onnx-ml.html

```python
# Train the model with CatBoost:
import catboost
from sklearn import datasets


iris = datasets.load_iris()
model = catboost.CatBoostClassifier(loss_function='MultiClass')

model.fit(iris.data, iris.target, verbose=False)

# Save model to ONNX-ML format
model.save_model(
    "iris.onnx",
    format="onnx",
    export_parameters={
        'onnx_domain': 'ai.catboost',
        'onnx_model_version': 1,
        'onnx_doc_string': 'test model for MultiClassification',
        'onnx_graph_name': 'CatBoostModel_for_MultiClassification'
    }
)

# Apply the model with onnxruntime:

import numpy as np
from sklearn import datasets
import onnxruntime as rt


iris = datasets.load_iris()

sess = rt.InferenceSession('iris.onnx')

# can get only label
label = sess.run(['label'],
                 {'features': iris.data.astype(np.float32)})

# can get only probabilities
probabilities = sess.run(['probabilities'],
                         {'features': iris.data.astype(np.float32)})

# or both
label, probabilities = sess.run(['label', 'probabilities'],
                                {'features': iris.data.astype(np.float32)})
```

Gives the following traceback:

```
Traceback (most recent call last):
  File "test-onnx-multi.py", line 32, in <module>
    sess = rt.InferenceSession('iris.onnx')
  File "/home/ei-grad/.virtualenvs/catboost37/lib/python3.7/site-packages/onnxruntime/capi/session.py", line 25, in __init__
    self._load_model(providers)
  File "/home/ei-grad/.virtualenvs/catboost37/lib/python3.7/site-packages/onnxruntime/capi/session.py", line 43, in _load_model
    self._sess.load_model(providers)
onnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization: /onnxruntime_src/onnxruntime/core/providers/cpu/ml/tree_ensemble_classifier.cc:294 void onnxruntime::ml::TreeEnsembleClassifier<T>::Initialize() [with T = float] base_values_.empty() || base_values_.size() == static_cast<size_t>(class_count_) || base_values_.size() == weights_classes_.size() was false. 
```

Binary classification example - works.

**Comment by annaveronika** [comment]
Thanks for pointing this out! We'll look into this.

**Comment by andrey-khropov** [comment]
This issue has been fixed by commit 6334602bcb68c18f7c7ee9f3c3bce452b628df4b and the fix will be included in the next release.
